ydb/docs/en/core/reference/ydb-cli/workload-tpcds.md

```diff
@@ -0,0 +1,96 @@
+# TPC-DS workload
+
+The workload is based on the TPC-DS [documentation](https://www.tpc.org/TPC_Documents_Current_Versions/pdf/TPC-DS_v3.2.0.pdf), and the queries and table schemas are adapted for {{ ydb-short-name }}.
+
+The test generates a typical workload in the decision support area.
+
+## Common command options
+
+All commands support the common option `--path`, which specifies the path to the directory with tables in the database:
+
+```bash
+{{ ydb-cli }} workload tpcds --path tpcds/s1 ...
+```
+
+### Available options { #common_options }
+
+Option name | Option description
+---|---
+`--path` or `-p` | Path to the directory with tables. Default value is `/`
+
+## Initializing the load test {#init}
+
+Before running the benchmark, create a table:
+
+```bash
+{{ ydb-cli }} workload tpcds --path tpcds/s1 init
+```
+
+See the description of the command to run the load:
+
+```bash
+{{ ydb-cli }} workload tpcds init --help
+```
+
+{% include [init_options](./_includes/workload/init_options_tpc.md) %}
+
+## Loading data into the table { #load }
+
+Load data into the table. The data will be generated directly by {{ ydb-cli }}:
+
+```bash
+{{ ydb-cli }} workload tpcds --path tpcds/s1 import generator --scale 1
+```
+
+See the description of the command to load the data:
+
+```bash
+{{ ydb-cli }} workload tpcds import --help
+```
+
+### Available options {#load_files_options}
+
+Option name | Option description
+---|---
+`--scale <value>` | Data scale. Powers of ten are usually used.
+`--tables <value>` | Comma-separated list of tables to generate. Available tables: `customer`, `nation`, `order_line`, `part_psupp`, `region`, `supplier`. All tables are created by default.
+`--proccess-count <value>` or `-C <value>` | Data generation can be split into several processes, this parameter specifies the number of processes. Default value: 1.
+`--proccess-index <value>` or `-i <value>` | Data generation can be split into several processes, this parameter specifies the process number. Default value: 0.
+`--state <path>` | Path to the download state file. If the download was interrupted for some reason, the download will be continued from the same place when it is started again.
+`--clear-state` | Relevant if the `--state` parameter is specified. Clear the state file and start the download from the beginning.
```

Comment:
```suggestion
| Name                                  | Description                                                                                                                                                    | Default value |
|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
| `--scale <value>`                     | Data scale. Typically, powers of ten are used.                                                                                                                  |               |
| `--tables <value>`                    | Comma-separated list of tables to generate. Available tables: `customer`, `nation`, `order_line`, `part_psupp`, `region`, `supplier`.                           | All tables    |
| `--process-count <value>` or `-C <value>` | Specifies the number of processes for parallel data generation.                                                                                                  | `1`           |
| `--process-index <value>` or `-i <value>` | Specifies the process number when data generation is split into multiple processes.                                                                              | `0`           |
| `--state <path>`                      | Path to the state file for resuming downloads. If the download is interrupted, it will resume from the same point when restarted.                                |               |
| `--clear-state`                       | Relevant if the `--state` parameter is specified. Clears the state file and restarts the download from the beginning.                                             |               |
```